root-config/src/index.ejs

```diff
@@ -78,7 +78,8 @@
           "@app/react-apps/common-footer-app": "//localhost:5005/app-common-hero-footer-app.js",
           "@app/react-apps/main-featured-product": "//localhost:9003/app-main-featured-product.js",
           "@app/angular-apps/common-header-app": "//localhost:5001/main.js",
-          "@app/angular-apps/contact-form-app": "//localhost:6003/main.js"
+          "@app/angular-apps/contact-form-app": "//localhost:6003/main.js",
+          "@app/main-categories-app": "//localhost:9002/main.js"
```

Comment:
I think it should be: "@app/angular-apps/main-categories-app": "//localhost:9002/main.js"